shell/platform/darwin/macos/framework/Source/FlutterTextInputPluginTest.mm

```diff
@@ -618,7 +595,7 @@ - (bool)testAutocompleteDisabledWhenAutofillGroupIncludesPassword {
   return true;
 }
 
-- (bool)testContentTypeWhenAutofillTypeIsUsername {
+- (bool)testContentTypeWhenAutofillGroupIncludesPassword {
```

Comment:
It loooks like this is two tests: username content type, and `obscureText` config. Should we split these up into two separate tests? I don't feel strongly either ways.

Reply:
I think the diff was playing tricks here due to the deleted test right next to it; after restoring and re-pushing it looks right to me. The test without obscureTest checks the content type is username. The one with obscureText checks password autofill type.